docs/tutorials/authentication.md

```diff
@@ -33,47 +34,83 @@ Welcome, First Last!
 Logged in as: username (1234567)
 ```
 
-## Use `.synapseConfig`
+## Use `.synapseConfig` for Multiple Profiles
 
 For writing code using the Synapse Python client that is easy to share with others, please do not include your credentials in the code. Instead, please use the `~/.synapseConfig` file to manage your credentials.
 
+The Synapse Python Client supports multiple profiles within the ~/.synapseConfig file, enabling users to manage credentials for multiple accounts.
+
+Each profile is defined in its own '[profile <profile_name>]' section. A default profile can still be defined using '[default]'.
+
 When installing the Synapse Python client, the `~/.synapseConfig` is added to your home directory.
 
 ### Automatically modifying the `~/.synapseConfig` file with the Command line Client
 You may modify the `~/.synapseConfig` file by utilizing the [command line client command and following the interactive prompts](./command_line_client.md#config):
 
 <!-- termynal -->
 ```
-> synapse config
-Synapse username (Optional):
+#For Default profile
+> synapse --profile default config
+Synapse username (Optional): $MY_USERNAME
+
+Auth token: $MY_SYNAPSE_TOKEN
+
+#For additional profiles
```

Comment:
similarly, I think here you could say something like: "for adding a new profile with a new username"